meson.build

```diff
@@ -731,9 +718,8 @@ conf.set('TTY_GID', tty_gid)
 substs.set('TTY_GID', tty_gid)
 
 # Ensure provided GID argument is numeric, otherwise fallback to default assignment
-if get_option('users-gid') != ''
-        users_gid = get_option('users-gid').to_int()
-else
+users_gid = get_option('users-gid')
+if users_gid == -1
         users_gid = '-'
```

Comment:
Pre-existing... But this will end up being an int or '-'... I guess that's fine for now, but perhaps keeping -1 would be better?